src/BaseTemplates/EmptyWeb/EmptyWeb.csproj

```diff
@@ -8,10 +7,24 @@
     <RuntimeIdentifier>win7-x86</RuntimeIdentifier>$endif$
   </PropertyGroup>
 
+  <PropertyGroup>  
+    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;node_modules\**;jspm_packages\**;bower_components\**;**\*.user;**\*.*proj</GlobalExclude>
+  </PropertyGroup>
+
   <ItemGroup>
+    <None Include="**\*" />
     <Compile Include="**\*.cs" />
     <EmbeddedResource Include="**\*.resx" />
+    <Content Include="wwwroot\**" CopyToPublishDirectory="PreserveNewest" />
+    <Content Include="**\*.cshtml" Exclude="wwwroot\**\*.cshtml" CopyToPublishDirectory="PreserveNewest" />
+    <Content Include="**\*.config" Exclude="wwwroot\**\*.config" CopyToPublishDirectory="PreserveNewest" />
+    <Content Include="**\*.json" Exclude="wwwroot\**\*.json" CopyToPublishDirectory="PreserveNewest" />
     <Folder Include="wwwroot\" />
+
+    <None Remove="@(Content);@(Compile);@(EmbeddedResource);$(GlobalExclude)" />
```

Comment:
@davidfowl the only issue with moving these to the targets is that CPS will no longer know where to put changes that override them, but this can be worked around if the user changes their csproj to remove the item from the item group & put it in to the right one before the targets run